integration/actions_test.go

```diff
@@ -29,9 +29,9 @@ import (
 	"github.com/TBD54566975/scaffolder"
 )
 
-// scaffold a directory relative to the testdata directory to a directory relative to the working directory.
-func scaffold(src, dest string, tmplCtx any) action {
-	return func(t testing.TB, ic testContext) error {
+// Scaffold a directory relative to the testdata directory to a directory relative to the working directory.
```

Comment:
Can't see it here, but the package is currently named `simple_test`. The files will have to be renamed to drop `_test.go` (except `integration_test.go`), the public API will have to be renamed to `integration`.

The `integration_test.go` file should be in the package `integration_test`.